app/components/blacklight/search_context/server_item_pagination_component.rb

```diff
@@ -12,35 +12,48 @@ def initialize(search_context:, search_session:, current_document:)
       end
 
       def render?
-        @search_context.present? && (@search_context[:prev] || @search_context[:next]) && (@search_session['document_id'] == @current_document_id)
+        return false unless @search_session['document_id'] == @current_document_id
+
+        return true if total == 1
+
+        @search_context.present? && (@search_context[:prev] || @search_context[:next])
```

Comment:
is this effectively the same as:
```suggestion
        @search_context.present?
```

Reply:
Right – my guess is that case (you have a context, but no prev or next) is the same thing as having a single result. But just a guess!

Reply:
Separately I have to remove the "|" separating the different sections, so that will be another commit.